### PR TITLE
fix: pass retry step to the backend

### PR DESCRIPTION
**Pull request by edewit**
this should be used in together with https://github.com/fabric8-launcher/ngx-launcher/pull/277

### CODE REVIEW

**Comment by joshuawilson** [comment]
please resolve conflicts
Please remove auto-upgrade caret

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @edewit for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by joshuawilson** [comment]
please update and add test cases where needed

**Comment by joshuawilson** [comment]
I can't get the tests to pass locally. Did you run `npm run test:unit`?

**Comment by invincibleJai** [comment]
@edewit Thanks looks good, tried with fabric8-launcher/ngx-launcher#277. Tests are failing that needs to be fixed and need to update package.json with latest ngx-forge post fabric8-launcher/ngx-launcher#277 gets merged

**Comment by edewit** [comment]
@joshuawilson @invincibleJai done

**Comment by invincibleJai** [comment]
@edewit what is node and npm version you are using ? am on node version 8.3.0 and npm 5.3.0. 
which is recommended and getting below error : 

screen shot 2018-06-19 at 8 25 10 pm

File: package.json [hunk 1]
```diff
@@ -141,7 +141,7 @@
     "ngx-base": "2.3.2",
     "ngx-bootstrap": "1.9.3",
     "ngx-fabric8-wit": "6.20.4",
-    "ngx-forge": "0.2.34",
+    "ngx-forge": "^0.3.6",
```
Comment (invincibleJai) [inline package.json]: Do we need ^ here?

Comment (joshuawilson) [reply package.json]: we should not use the caret and you will need to update the package-lock again

**Comment by jyasveer** [comment]
@joshuawilson can you please merge this. Thanks

**Pull request merged**